nodes/nodes-about-autoscaling-nodes.adoc

```diff
@@ -0,0 +1,78 @@
+include::modules/attributes-openshift-dedicated.adoc[]
+[id="nodes-about-autoscaling-nodes"]
+= About autoscaling nodes on a cluster
+:context: nodes-about-autoscaling-nodes
+toc::[]
+
+ifdef::openshift-dedicated[]
+[IMPORTANT]
+====
+Autoscaling is only available on clusters that were purchased through the Red Hat Marketplace.
+====
+endif::[]
+
+The autoscaler option can be configured to automatically scale the number of machines in a cluster.
+
+The cluster autoscaler increases the size of the cluster when there are pods that failed to schedule on any of the current nodes due to insufficient resources or when another node is necessary to meet deployment needs. The cluster autoscaler does not increase the cluster resources beyond the limits that you specify.
+
+Additionally, the cluster autoscaler decreases the size of the cluster when some nodes are consistently not needed for a significant period, such as when it has low resource use and all of its important pods can fit on other nodes.
+
+When you enable autoscaling, you must also set a minimum and maximum number of worker nodes.
+
+[NOTE]
+====
+Only cluster owners and organization admins can scale or delete a cluster.
+====
+
+[id="nodes-enabling-autoscaling-nodes"]
+== Enabling autoscaling nodes on a cluster
+
+Autoscale worker nodes on a cluster to increase or decrease the number of resources available. Autoscaling for worker nodes is configured in the machine pool definition if you are configuring an existing cluster.
+
+[discrete]
+include::modules/ocm-enabling-autoscaling-nodes.adoc[leveloffset=+2]
+
+ifdef::openshift-rosa[]
+[NOTE]
+====
+Additionally, you can configure autoscaling on the default machine pool when you xref:../rosa_getting_started/rosa-creating-cluster.adoc#rosa-creating-cluster[create the cluster using interactive mode].
+====
+
+[discrete]
+include::modules/rosa-enabling-autoscaling-nodes.adoc[leveloffset=+2]
```

Comment:
In this file, I believe, there is a line: _Decide which machine pool ID you want to configure._
That suggests to me that you can configure autoscaling on only one machine pool ID. If not, maybe make it _machine pool IDs?

If I want to be especially picky, I might say change this to: _Get the ID of the machine pools that you want to confgure._ ;-)

Reply:
If I want to be especially picky, I might say change this to: _Get the ID of the machine pools that you want to confgure._